src/components/file-selector/file-selector.scss

```diff
@@ -64,12 +65,14 @@
   @include sbb.text--bold;
   @include sbb.title-6($exclude-spacing: true);
 
+  text-align: center;
```

Comment:
Also, the text-align does not seem to have any effect. Am I missing something?

Reply:
From figma, for long subtitle, it should align to center and not on left side